tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -222,9 +202,14 @@
         // Which tab is selected (scalars, graph, images etc).
         mode: {
           type: String,
-          computed: '_getModeFromIndex(modeIndex)',
+          computed: '_getModeFromIndex(tabs, modeIndex)',
```

Comment:
Actually, maybe we can replace the "tab" name with "plugin name" (pluginNames in this case) across the board.